gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/README.adoc

```diff
@@ -0,0 +1,252 @@
+= Gravitee.io API Management - Gateway Testing SDK
+
+ifdef::env-github[]
+image:https://ci.gravitee.io/buildStatus/icon?job=gravitee-io/gravitee-gateway/master["Build status", link="https://ci.gravitee.io/job/gravitee-io/job/gravitee-gateway"]
+image:https://f.hubspotusercontent40.net/hubfs/7600448/gravitee-github-button.jpg["Join the community forum", link="https://community.gravitee.io?utm_source=readme", height=20]
+endif::[]
```

Comment:
Useless

Reply:
removed in [e6cc8fb](https://github.com/gravitee-io/gravitee-api-management/pull/1939/commits/e6cc8fbc0d09d3672905c5bf316679747e91303d)